benchmarks/micro/requirements-compression-algorithms.txt

```diff
@@ -1,3 +1 @@
-zstandard
```

Comment:
Any reason this has been moved into dev-requirements?

Reply:
I would prefer to keep multiple requirements files unless there is a specific reason to only have one.

Reply:
Yeah, Sorry those changes didn't get there, they will be separated.

Reply:
Added multiple requirements files. Now there are only test and benchmark-related requirements, but there will be new ones in further PRs.